### PR TITLE
[TIMOB-24629] (6_2_X) Android: Add Notification wakeLock

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24629

### CODE REVIEW

**Review by garymathews — APPROVED** [review]
CR: **PASS**

👍

**Comment by lokeshchdhry** [comment]
@garymathews , Can you please resolve the conflicts. Thanks.

**Comment by lokeshchdhry** [comment]
@hansemannn , can you please resolve the conflicts so that I can merge this PR.

File: android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java [hunk 1]
```diff
@@ -128,11 +128,9 @@ public void handleCreationDict(KrollDict d)
 		if (d.containsKey(TiC.PROPERTY_GROUP_KEY)) {
 			setGroupKey(TiConvert.toString(d, TiC.PROPERTY_GROUP_KEY));
 		}
-		/*
 		if (d.containsKey(TiC.PROPERTY_GROUP_ALERT_BEHAVIOR)) {
 			setGroupAlertBehavior(TiConvert.toInt(d, TiC.PROPERTY_GROUP_ALERT_BEHAVIOR));
 		}
-		*/
```
Comment (hansemannn) [inline android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java]: @garymathews Just to be sure: This needs to be uncommented in 6_2_X? This was part of the merge-conflicts.

**Comment by garymathews** [comment]
@lokeshchdhry Fixed merge conflict

**Comment by lokeshchdhry** [comment]
Wake lock flags works as expected with appropriate wake lock level and flags.

Studio Ver:     4.9.1.201707200100
SDK Ver:        6.2.0 local build
OS Ver:         10.12.3
Xcode Ver:      Xcode 8.3.3
Appc NPM:       4.2.9
Appc CLI:       6.2.3
Ti CLI Ver:     5.0.14
Alloy Ver:      1.9.13
Node Ver:       6.10.1
Java Ver:       1.8.0_101
Devices:        ⇨ google Pixel --- Android 7.1.1
                      ⇨ google Nexus 5 --- Android 6.0.1
                      ⇨ Motorola G2 --- Android 4.4.4

**Pull request merged**